src/test/burn/WixToolsetTest.BurnE2E/CacheTests.cs

```diff
@@ -17,23 +18,35 @@ public class CacheTests : BurnE2ETests
         [Fact]
         public void CanCache5GBFile()
         {
-            var packageA = this.CreatePackageInstaller("PackageA");
-            var bundleC = this.CreateBundleInstaller("BundleC");
-
-            packageA.VerifyInstalled(false);
-
             // Recreate the 5GB payload to avoid having to copy it to the VM to run the tests.
+            const long FiveGB = 5_368_709_120;
+            const long OneGB = 1_073_741_824;
             var targetFilePath = Path.Combine(this.TestContext.TestDataFolder, "fivegb.file");
+
+            // If the drive has less than 5GB (for the test file) plus 1GB (for working space), then
+            // skip the test.
+            var drive = new DriveInfo(targetFilePath.Substring(0, 1));
+            if (drive.AvailableFreeSpace < FiveGB + OneGB)
+            {
+                Console.WriteLine("Skipping CanCache5GBFile() test because there is not enough disk space available to run the test.");
```

Comment:
The 5GB file was created while building all of the test bundles and then deleted afterwards, so there's pretty much a 0% chance of this occurring. A similar check would needed to be added to skip creating the file and building the bundle for this to be effective.

Reply:
I had test runs fail until I added this code. This was the last fix needed to get everything passing.

Reply:
Maybe all the debugging log statements filled up the hard drive? :)